Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: When a checkbox is marked as `required`, it must be checked to be valid. Per the instructions, the user only needs to choose one or more, so `required` must not be used on any of the checkbox inputs.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
diff --git a/output/Checkbox Group with Validation Error/2/a255cfeb-1feb-4b0b-bbbb-b8e971d0da95.html b/output/Checkbox Group with Validation Error/2/a255cfeb-1feb-4b0b-bbbb-b8e971d0da95.html
index 243cd33..62a4a60 100644
--- a/output/Checkbox Group with Validation Error/2/a255cfeb-1feb-4b0b-bbbb-b8e971d0da95.html	
+++ b/output/Checkbox Group with Validation Error/2/a255cfeb-1feb-4b0b-bbbb-b8e971d0da95.html	
@@ -1,16 +1,16 @@
-<fieldset>
+<fieldset aria-describedby="fruitsError">
   <legend>What fruits do you like?</legend>
+  <span id="fruitsError">You must choose one or more fruits</span>
   <div>
-    <input type="checkbox" id="bananas" name="fruits" value="bananas" required>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas" aria-invalid="true">
     <label for="bananas">Bananas</label>
   </div>
   <div>
-    <input type="checkbox" id="oranges" name="fruits" value="oranges">
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges" aria-invalid="true">
     <label for="oranges">Oranges</label>
   </div>
   <div>
-    <input type="checkbox" id="apples" name="fruits" value="apples">
+    <input type="checkbox" id="apples" name="fruits[]" value="apples" aria-invalid="true">
     <label for="apples">Apples</label>
   </div>
-  <span style="color: red; display: none;" id="fruitsError">You must choose one or more fruits</span>
 </fieldset>
\ No newline at end of file
